web/src/beta/lib/core/utils/ImageryLayer.ts

```diff
@@ -0,0 +1,11 @@
+import { ImageryLayer as CesiumImageryLayer, type ImageryProvider } from "@cesium/engine";
```

Comment:
Could you put this definitions into `/JapanGSIOptimalBVmapImageryLayer/types.ts`